### PR TITLE
fix(Modal): Fixed bottom padding on the Modal header

### PR DESCRIPTION
**Pull request by chris-cedrone-cengage**
Issue: # [1173](https://github.com/cengage/react-magma/issues/1173)

## What I did
Removed bottom padding from `Modal` header.

## Screenshots:
Paddingery

## Checklist 
- [X] changeset has been added
- [X] Pull request description is descriptive
- [N/A] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes
- [N/A] I have added tests that prove my fix is effective or that my feature works

## How to test
- Go to Doc site
- Open Modal and ensure there is no bottom padding on the `Modal` header

### CODE REVIEW

File: .changeset/fix-modalPadding.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'react-magma-dom': minor
```
Comment (silvalaura) [inline .changeset/fix-modalPadding.md:2]: patch

**Review by silvalaura — APPROVED** [review]

**Comment by silvalaura** [comment]
A couple things I noticed for discussion:
1. The example for modal with no header looks odd:
image
2. The modal with header with only one paragraph (no margins) also looks odd:
image
3. Hide close button, no margin:
image
4. In the nested modal example, the second modal's paragraph needs `noMargins` 
image


I guess the question is, are we okay with no spacing between the button and paragraph in these scenarios?
@orion-cengage

**Review by orion-cengage — APPROVED** [review]

**Comment by silvalaura** [comment]
This PR is on hold until https://github.com/cengage/react-magma/pull/1177 gets merged so we can use the new props here.

File: website/react-magma-docs/src/pages/api/modal.mdx [hunk 2]
```diff
@@ -295,7 +315,7 @@ export function Example() {
         }}
         isOpen={showModal}
       >
-        <p>This is a modal, doing modal things.</p>
+        <Paragraph noMargins>This is a modal, doing modal things.</Paragraph>
```
Comment (silvalaura) [inline website/react-magma-docs/src/pages/api/modal.mdx:318]: image

```suggestion
        <Paragraph noTopMargin>This is a modal, doing modal things.</Paragraph>
```

File: website/react-magma-docs/src/pages/api/modal.mdx [hunk 3]
```diff
@@ -343,24 +364,24 @@ export function Example() {
         isOpen={showModal}
         isInverse
       >
-        <p>This is a modal, doing modal things.</p>
-        <p>
+        <Paragraph noMargins>This is a modal, doing modal things.</Paragraph>
```
Comment (silvalaura) [inline website/react-magma-docs/src/pages/api/modal.mdx:367]: We need to add `isInverse` to all these Paragraphs
image

**Comment by silvalaura** [comment]
Just one more thing, I think at standup we talked about adding to storybook some of the similar examples to those in the docs (no header, small, no close button, etc)

**Comment by ccedrone** [comment]
> Just one more thing, I think at standup we talked about adding to storybook some of the similar examples to those in the docs (no header, small, no close button, etc)

There's an example without a header or close button in one, should there be more than that? I'll add the small example!

**Comment by silvalaura** [comment]
> https://storybook-preview-1175--upbeat-sinoussi-f675aa.netlify.app

I missed that, but that is good enough for me!

**Review by silvalaura — APPROVED** [review]
Looks good (again!). Just same comment about the PR title :)

**Pull request merged**